### PR TITLE
Several spelling suggestions

### PR DESCRIPTION
**Pull request by mdeweerd**
A fex spelling suggestions for the dictionnary.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -2614,6 +2614,7 @@ agani->again
 aganist->against
 agant->agent
 agants->agents, against,
+agein->again
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:2617]:
```suggestion
agein->again, ageing,
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -9735,6 +9736,7 @@ boostrappers->bootstrappers
 boostrapping->bootstrapping
 boostraps->bootstraps
 booteek->boutique
+bootime->boottime
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:9739]: Is that word? Perhaps `boot time` or `boot-time` instead?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -49546,6 +49551,7 @@ repostiories->repositories
 repostiory->repository
 repostories->repositories
 repostory->repository
+repot->report, repost,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:49554]: At least move to _rare_, `repot` is a word.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (mdeweerd) [reply codespell_lib/data/dictionary.txt:2617]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (mdeweerd) [reply codespell_lib/data/dictionary.txt:9739]: I made the suggestion two words.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (mdeweerd) [reply codespell_lib/data/dictionary.txt]: Moved to rare.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
Thank you @mdeweerd.

**Pull request merged**